File: problems/h/herman/herman.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>
#include <math.h>

double calc_euclidian_area(int radius) {
    return M_PI * pow(radius, 2);
}


double calc_taxicab_area(int radius) {
    return 2.0 * pow(radius, 2);
}


int main(void) {
    int r;

    if (scanf("%d", &r) != 1) {
        printf("Failed to read r!");
    }
    
    printf("%lf\n", calc_euclidian_area(r));
    printf("%lf\n", calc_taxicab_area(r));

    return 0;
}

File: problems/h/herman/herman.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

from math import pi

def calc_euclidian_area(radius):
    return pi * (radius ** 2)


def calc_taxicab_area(radius):
    return 2.0 * (radius ** 2)


def main():
    r = int(input())

    print(calc_euclidian_area(r))
    print(calc_taxicab_area(r))


if __name__ == '__main__':
    main()